### ISSUE TITLE
Group selection checkbox is missing aria-label

### ISSUE THREAD

**Issue by oleksandr-bilyk** [microsoft/fluentui#13385]
Hi. There is problem to set selection aria label. 
IGroup interface does not have property to set alia level on group selection checkbox.

https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist/grouped

![image](https://user-images.githubusercontent.com/17872879/83169530-12c78800-a10b-11ea-91f5-e15aa25eeff9.png)

**Comment by aneeshack4** [comment]
Hi @oleksandr-bilyk, I do repro this. 
![image](https://user-images.githubusercontent.com/4161791/83279253-764cc680-a19a-11ea-8f10-698553ef2484.png)
Thanks for reaching out to us. @ecraig12345 this might be a quick fix... what do you think?

**Comment by ecraig12345** [comment]
It looks like we don't have a prop to set the group header checkbox aria label, surprisingly. In theory you can set it using `IGroupedListProps.headerProps.selectAllButtonProps.ariaLabel`, except there's not a way to pass that through via DetailsList. Definitely something we should add (and update the examples), though figuring out the "right" way to add it may be a bit complicated--looking through DetailsList I see multiple existing patterns for setting aria labels on other checkboxes.

Tentatively I'd suggest adding a prop called `IDetailsListProps.ariaLabelForGroupCheckbox` which is passed through to GroupedList as mentioned above, but I'm not 100% sure about the naming or location. @dzearing or @ThomasMichon (since you've worked some on DetailsList) any thoughts?

**Comment by JustSlone** [comment]
+1 on this from internal partner



Path | div[data-focuszone-id="FocusZone91"] > .ms-GroupHeader-check.check-370[role="checkbox"]
-- | --
Snippet | `<button type="button" class="ms-GroupHeader-check check-370" role="checkbox" aria-checked="false" data-selection-toggle="true" tabindex="-1">`

Fix ONE of the following:  
- Element does not have inner text that is visible to screen readers
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element's default semantics were not overridden with role="presentation" 
- Element's default semantics were not overridden with role="none" 
- Element has no title attribute or the title attribute is empty

**Comment by premchandpl76** [comment]
selectAllButtonProps={{ '**aria-label**': 'Group checkbox' }} has solved the issue for me

**Comment by ecraig12345** [comment]
Still a valid issue
